.github/workflows/stats_merged_prs.yml

```diff
@@ -0,0 +1,24 @@
+name: Merged PR stats
+
+on:
+  pull_request_target:
+    branches:
+      - main
+      - v*-branch
+    types: [closed]
+jobs:
+  record_merged:
+    if: github.event.pull_request.merged == true && github.repository == 'zephyrproject-rtos/zephyr'
+    runs-on: ubuntu-22.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
```

Comment:
Note that the first run of this workflow is going to fail because it will check out the base branch (`main`) instead of the pull request head (`github.event.pull_request.head.sha`) -- that is a good thing because it does not expose `ELASTICSEARCH_KEY` wide open if someone decides to change `types: [closed]` to something else in the future.